pubspec.yaml

```diff
@@ -4,28 +4,28 @@ description: General utilities for Dart projects.
 homepage: https://www.github.com/Workiva/w_common
 
 environment:
-  sdk: '>=2.4.0 <3.0.0'
+  sdk: '>=2.9.0 <3.0.0'
```

Comment:
Why `2.9.0` and not `2.12.0`?

Reply:
If we say 2.12.0 is the minimum, that opts all the code into null-safety by default, and I didn't try to do that conversion. So 2.9.0 is to prevent it being picked up from 2.7.2 users.

Reply:
If it's 2.12 then it forcibly opts in all the code to null-safety. I didn't want to do that conversion as well, so 2.9 is high enough to stop 2.7.2 clients from using it, where it won't work.